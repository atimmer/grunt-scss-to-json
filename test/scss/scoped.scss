%scoped {
  $first: 15px;
  $global-variable: $first + 2 !global;
  $color: #123;
  $global-with-function: quote($color) !global;
}

html {
  @extend %scoped;
}
