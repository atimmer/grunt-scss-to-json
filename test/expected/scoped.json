{"$global-variable":"17px","$global-with-function":"#0b1520"}